tests/models/test_variational_gp.py

```diff
@@ -61,5 +63,52 @@ def test_1d_classification(self):
         npt.assert_array_less(1, pv)
 
 
+class AxOrdinalGPTestCase(unittest.TestCase):
+    @classmethod
+    def setUp(cls):
+        np.random.seed(1)
+        torch.manual_seed(1)
+        cls.n_levels = 5
+        X, y = make_classification(
+            n_samples=20,
+            n_features=5,
+            n_classes=cls.n_levels,
+            n_informative=3,
+            n_clusters_per_class=1,
+        )
+        cls.X, cls.y = torch.Tensor(X), torch.Tensor(y).reshape(-1, 1)
+
+    def test_ordinal_classification(self):
```

Comment:
Since this test passes, does that assuage your worries about only get 0/4 predictions before?

Reply:
yes, it does.